receiver/iisreceiver/testdata/scraper/expected.json

```diff
@@ -281,14 +326,29 @@
                         "dataPoints": [
                            {
                               "asInt": "1",
-                              "startTimeUnixNano": "1649969943717902800",
-                              "timeUnixNano": "1649969943717902800"
+                              "startTimeUnixNano": "1664375532831495700",
+                              "timeUnixNano": "1664375532831495700"
                            }
                         ],
                         "isMonotonic": true
                      },
                      "unit": "{requests}"
-                  },
+                  }
+               ],
+               "scope": {
+                  "name": "otelcol/iisreceiver",
+                  "version": "latest"
+               }
+            }
+         ]
+      },
+      {
+         "resource": {
+            "attributes": []
```

Comment:
Just as a sanity check, this resource represents the IIS server as a whole, and the threads metric should stay with this resource. Is that right?

Reply:
Yep, that's correct. The counter it scrapes is per-process, and there's no clear way to associate sites or app pools with specific processes, so I've left this counter as-is.